docs/data/charts/heatmap/heatmap.md

```diff
@@ -70,7 +70,23 @@ You can import the default tooltip, or only its content as follows:
 import { HeatmapTooltip, HeatmapTooltipContent } from '@mui/x-charts/Heatmap',
 ```
 
-## Legend 🚧
+The Heatmap has an item tooltip that can be customized as described in the [Tooltip documentation page](/x/react-charts/tooltip/).
+
+The specificity of the Heatmap Tooltip is its default content.
+You can import the default tooltip, or only its content as follow:
+
+```js
+import { HeatmapTooltip, HeatmapTooltipContent } from '@mui/x-charts/Heatmap',
+```
+
+### Legend
+
+The Heatmap comes with a legend which is by default the [ContinuouseColorLegend](/x/react-charts/legend/#color-legend).
+
+To display it set `hideLegend` to `false`.
```

Comment:
I suppose we default to not showing the legend to not break older versions. But should we default to always show it instead? 🤔

Reply:
Effectively in v9 the default should be revert

Reply:
Do we have an umbrella issue for the changes for v9? It'd probably be useful so we don't forget everything we need to do then.